pages/common/tree.md

```diff
@@ -14,14 +14,14 @@
 
 `tree -a`
 
-- Print human readable size of files:
+- Print the tree without indentation lines, showing the full path instead:
 
-`tree -h`
+`tree -i -f`
 
-- Print the full path for each file:
+- Print the size of each node next to it, in human-readable format, with folders displaying their cumulative size (as in the `du` command):
```

Comment:
We could remove the --du option, but that makes the output less practical. I wonder if what makes it seem complex is the length of the description. Would something like this be better?

"Print the human-readable size of each node next to it (with folders displaying their cumulative size, as in the `du` command):"